### ISSUE TITLE
cudnn and pytorch 1.0

### ISSUE THREAD

**Issue by bloodymeli** [Microsoft/human-pose-estimation.pytorch#67]
Hi,

Is there a way to disable cudnn for batch norm on pytorch 1.0?

**Comment by leoxiaobin** [comment]
You can try pytorch 1.0 with cudnn enabled, it seems that pytorch 1.0 has fixed the bug.
